dom.bs

```diff
@@ -6697,23 +6715,19 @@ method, when invoked, must run these steps:
 
 <pre class=idl>
 [Exposed=Window]
-interface Attr {
+interface Attr : Node {
   readonly attribute DOMString? namespaceURI;
   readonly attribute DOMString? prefix;
   readonly attribute DOMString localName;
```

Comment:
Actually, it seems WebKit is the odd one here. Both Blink and Firefox have these attributes on Attr / Element instead of Node. So I guess I'll try to align WebKit.